scipy/stats/stats.py

```diff
@@ -6673,7 +6692,7 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     and expected frequencies should be at least 5. If one or more frequencies
     are less than 5, Fisher's Exact Test can be used with greater statistical
     power. According to [3]_, the total number of samples is recommended to be
-    greater than 13, otherwise a table-based method of obtaining p-values is 
+    greater than 13, otherwise a table-based method of obtaining p-values is
```

Comment:
These docs have been changed in gh-14012. If you revert this change, I think the conflicts should go away:

```suggestion
    greater than 13, otherwise a table-based method of obtaining p-values is 
```